### DISCUSSION TITLE
YT videos download now in the first translated language listed (German) no matter the produced video (English original)

### DISCUSSION THREAD

**Discussion by commerc** [aclap-dev/video-downloadhelper#1563] [Help and Q&A]
### Your question

Change in download behaviour due to a YT update. Original language of video and played language is ignored, and downloads are in the first-listed (auto translate) language. The list of options begins with German, so any video is dowloaded in German.

### Addon & CoApp version and details

Change in download behaviour due to a YT update. Original language of video and played language is ignored, and downloads are in the first-listed (auto translate) language. The list of options begins with German, so any video is dowloaded in German.

**Comment by GrampaWildWilly** [comment]
This has been reported before.  Search this forum to find other discussions of this topic.
